src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block">
+      <div class="stars stars--0">
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+      </div>
+      <div class="stars  stars--1">
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+      </div>
+      <div class="stars  stars--2">
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+        <div class="strars__star"></div>
+      </div>
+      <div class="stars  stars--3">
```

Comment:
```suggestion
      <div class="stars stars--3">
```

remove one redundant space please